### ISSUE TITLE
s3-post-policy doesn't support range policies

### ISSUE THREAD

**Issue by jbuck** [jbuck/s3-post-policy#1]
For example, if you pop `["content-length-range", 1048579, 10485760]` into the policy, you'll get unexpected results when trying to submit that to S3. See http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-HTTPPOSTConstructPolicy.html for more details.

**Comment by ghost** [comment]
Any timeline on a fix for this?

**Comment by ghost** [comment]
I modified this part of the file so that it doesn't error out if you use a number in the second field

``` javascript
 policy.conditions.forEach(function(p) {
    if (Array.isArray(p)) {
      if (p[0] === "starts-with") {
        output.starts_with.push(p[1].substr(1));
      }
      if(isNaN(p[1]))   //new
        output.fields[p[1].substr(1)] = p[2];
    } else {
      var key = Object.keys(p)[0];
      output.fields[key] = p[key];
    }
  });
```

**Comment by jbuck** [comment]
@dtruel Hi there! I haven't worked on/used this module in more than a year. Let me take a look at this later this weekend, hopefully I can add some sort of test for this...

**Comment by ghost** [comment]
Great!  I should say thanks for publishing this module - made my life much easier not having to figure out how to mess with all the signing stuff!

**Comment by ayakovlenko** [comment]
@jbuck Thank you for this package. @dtruel's fix seems to do the trick. I can do a PR. Could you merge and release it?

**Comment by jbuck** [comment]
Fixed by #7
